.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I see another commit that removes this file. Make sure that you squash those commits because this accident should not be preserved in the repo history.